perfkitbenchmarker/providers/gcp/gce_virtual_machine.py

```diff
@@ -131,7 +190,11 @@ def _GetOptionDecoderConstructions(cls):
     """
     result = super(CustomMachineTypeSpec, cls)._GetOptionDecoderConstructions()
     result.update({'cpus': (option_decoders.IntDecoder, {'min': 1}),
-                   'memory': (MemoryDecoder, {})})
+                   'memory': (MemoryDecoder, {}),
+                   'gpus': (GceGpuDecoder, {
```

Comment:
Would it be significantly harder to just move the GpuDecoder to the GceVmSpec rather than the custom machine type spec? Seems like that would have the desired effect with just a couple of small changes. If not, implementing it later works for me